lib/session.js

```
@@ -478,198 +490,183 @@ function checkLongSmses() {
  * Generic session function
  *
- * @param {object} options - {sock, log}
- * @return {object} (returnObj)
+ * @param obj sock - socket object
+ * @return obj (returnObj)
  */
-function session(options) {
-	const	logPrefix	= topLogPrefix + 'session() - socket address: ' + options.sock.remoteAddress + ':' + options.sock.remotePort + ' - ',
-		returnObj	= new events.EventEmitter();
+function session(sock) {
+	var returnObj = new events.EventEmitter();
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)